spec/system/posts_index_spec.rb

```diff
@@ -68,7 +63,7 @@
   # I can see the first comments on a post
   it 'can see the first comments of a post' do
     posts.each do |post|
-      post.five_most_recent_comments.each do |comment|
+      post.comments.each do |comment|
         expect(page).to have_content(comment.text)
       end
     end
```

Comment:
- [ ] @Team Great work adding tests for the posts' index page, however on this test, you are supposed to check if the `five_most_recent_comments` method is working correctly, that given more than five comments it will display five recent comments.(You can use the example in the users show page on how to do it)